insights/specs/insights_archive.py

```diff
@@ -106,7 +106,6 @@ class InsightsArchiveSpecs(Specs):
     ipv4_neigh = simple_file("insights_commands/ip_-4_neighbor_show_nud_all")
     ipv6_neigh = simple_file("insights_commands/ip_-6_neighbor_show_nud_all")
     iscsiadm_m_session = simple_file("insights_commands/iscsiadm_-m_session")
-    keystone_crontab = simple_file("insights_commands/crontab_-l_-u_keystone")
```

Comment:
@ryan-blakley since you are changing the source of the data to be collected, we need to leave the archive spec in place, and change it to a `first_file` to ensure that it will be read from the archive regardless of whether the client collected it with the old spec or the new spec.

Reply:
ack updated